src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -252,95 +252,37 @@ class HeaderBar extends Component {
-                  )
-                  }
-                </Tabs>
-              </div>
-              {
-                voterIsSignedIn && (
-                  <div className="header-nav__avatar-wrapper u-cursor--pointer u-flex-none">
-                    {
-                      showEditAddressButton && (
-                        <Tooltip title="Change my location" aria-label="Change Address" classes={{ tooltipPlacementBottom: classes.tooltipPlacementBottom }}>
-                          <IconButton
-                            classes={{ root: classes.iconButtonRoot }}
-                            id="changeAddressHeaderBar"
-                            onClick={this.toggleSelectBallotModal}
-                          >
-                            <PlaceIcon />
-                          </IconButton>
-                        </Tooltip>
-                      )
-                    }
-                    {voterPhotoUrlMedium ? (
-                      <div id="profileAvatarHeaderBar"
-                           className={`header-nav__avatar-container ${isCordova() ? 'header-nav__avatar-cordova' : undefined}`}
-                           onClick={this.toggleProfilePopUp}
-                      >
-                        <img
-                          className="header-nav__avatar"
-                          src={voterPhotoUrlMedium}
-                          height={34}
-                          width={34}
-                          alt="Your Profile"
-                        />
-                      </div>
-                    ) : (
-                      <div>
-                        <IconButton
-                          classes={{ root: classes.iconButtonRoot }}
-                          id="profileAvatarHeaderBar"
-                          onClick={this.toggleProfilePopUp}
-                        >
-                          <AccountCircleIcon />
-                        </IconButton>
-                      </div>
-                    )
-                    }
-                    {this.state.profilePopUpOpen && voterIsSignedIn && (
-                      <HeaderBarProfilePopUp
-                        hideProfilePopUp={this.hideProfilePopUp}
-                        onClick={this.toggleProfilePopUp}
-                        profilePopUpOpen={this.state.profilePopUpOpen}
-                        signOutAndHideProfilePopUp={this.signOutAndHideProfilePopUp}
-                        toggleProfilePopUp={this.toggleProfilePopUp}
-                        toggleSignInModal={this.toggleSignInModal}
-                        transitionToYourVoterGuide={this.transitionToYourVoterGuide}
-                        voter={this.props.voter}
-                        weVoteBrandingOff={this.state.we_vote_branding_off}
-                      />
-                    )}
-                  </div>
+      <Wrapper hasNotch={hasIPhoneNotch()} scrolledDown={scrolledDown && isWebApp() && shouldHeaderRetreat(pathname)}>
```

Comment:
Hi @SailingSteve please let us know if you still need the div for finding this code:
div id="the next styled div is the Wrapper in HeaderBar"
It has been removed in this pull request from @lazorfuzz.